### ISSUE TITLE
Unable to see keyboard within Input list - Asus C302C Notebook PC

### ISSUE THREAD

**Issue by reallybigcatheter** [klausw/hackerskeyboard#607]
Hoping to utilize this great tool on my Chromebook platform *Dev mode of course) for developing in Python under Termux.

Followed install instructs, but keyboard does not appear in keyboard input list.

Cheers

**Comment by TPS** [comment]
https://github.com/klausw/hackerskeyboard/wiki/FrequentlyAskedQuestions#im-still-seeing-the-system-keyboard-how-do-i-activate-it

**Comment by reallybigcatheter** [comment]
Thanks, but been there, done that.  The keyboard does NOT show up in available keyboards.

**Comment by TPS** [comment]
When you go to Settings > Languages & Input > (Virtual) Keyboards, HK isn't listed anyplace?

**Comment by TPS** [comment]
There's also the note that [certain Android apps need additional permissions to work on Chromebooks](https://support.google.com/chromebook/answer/7021273?hl=en&ref_topic=3415446).

**Comment by reallybigcatheter** [comment]
Correct, Settings > Languages & Input > (Virtual) Keyboards does not display any HK entry.

Following the above link steps, Permissions is grayed out, so unable to make any changes.

Change app permissions
After you install an Android app, you can control which capabilities or information it has access to, known as permissions:

Click your account photo.
Click Settings Settings.
In the "Google Play Store" section, click Google Play Store.
Click Manage Android preferences.
In the "Device" section, click Apps.
Click an app in the list.
Click Permissions.  <<<<<  greyed out >>>>>>>>

Cheers
